src/components/Facets.tsx

```diff
@@ -95,6 +95,14 @@ export function Facets(props: FacetsProps) {
                 facetType = getFacetTypeFromFacet(facet, hierarchicalFieldIds);
               }
 
+              facetProps = {
+                customCssClasses: {
+                  ...customCssClasses,
+                  ...facetProps.customCssClasses,
+                },
+                ...facetProps
+              };
```

Comment:
is this line `...facetProps.customCssClasses,` necessary? It seems like if the `customFacetElement` has customCssClasses, then those will already take precedence over the customCssClasses defined here

Reply:
Ack. see other comment